src/realm.js

```diff
@@ -1737,12 +1743,31 @@ export class Realm {
     return !this._abstractValuesDefined.has(nameString);
   }
 
-  getTemporalBuildNodeEntryArgsFromDerivedValue(value: Value): void | TemporalBuildNodeEntryArgs {
+  getTemporalBuildNodeEntryFromDerivedValue(value: Value): void | TemporalBuildNodeEntry {
     let name = value.intrinsicName;
-    invariant(name);
+    if ((value instanceof AbstractValue && value.kind === "conditional") || !name) {
```

Comment:
Remove the `conditional` special case, I don't see how that's needed. If it is needed, let's talk about that.

Reply:
It's needed as `conditional` temporal entries get recorded too.

Reply:
I used to do this check in previous logic, but you said to not do a `conditional` check there either. The fact is, `conditional` generator entries do definitely get recorded.

Reply:
That should never happen. Repro?